web/styles.css

```diff
@@ -0,0 +1,105 @@
+:root {
+  --main-text: darkslategray;
+  --code-text: slategray;
+  --main-background: snow;
+  --link: hsl(14, 91%, 79%);
+  --link-visited: hsl(14, 51%, 74%);
+  --separator: rgb(150, 150, 150);
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  width: 800px;
+  margin: 0 auto;
+  padding-top: 2.5em;
+  padding-left: 0
+  padding-bottom: 5em;
+  padding-right: 0;
+  font-size: 18px;
+  color: var(--main-text);
+  background-color: var(--main-background);
+  line-height: 1.8em;
+}
+
+code {
+  color: var(--code-text);
+}
+
+@media screen and (max-width: 768px) {
+  body {
+    width: auto;
+    padding-top: 0.5em;
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+    padding-bottom: 3em;
+    font-size: 16px;
+  }
+
+  code {
+    width: auto;
+    display: block;
+    overflow: auto;
+  }
+}
+
+a {
+  color: var(--link);
+}
+
+a:visited {
+  color: var(--link-visited);
+}
+
+header {
+  margin-bottom: 1em;
+}
+
+header h1 {
+  margin: 0 0 0.3em;
+  padding: 0;
+  color: salmon;
+}
+
+header p {
+  margin: 0 0 1em;
+  padding: 0;
+  color: lightslategray;
+}
+
+nav ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+nav li {
+  display: inline;
+}
+
+nav li a {
+  text-decoration: none;
+}
+
+nav li a:hover {
+  text-decoration: underline;
+}
+
+nav li:after,
+.vertical-separator:after {
+  content: '/';
+  color: var(--separator);
+  margin-left: 0.7em;
+  margin-right: 0.4em;
+}
+
+nav li:last-child:after,
+.vertical-separator:last-child:after {
+  content: ''
```

Comment:
missing semi (technically is optional if it is the last rule in a block)

Not setting margin-left and margin-right to 0 here leaves them on the page.  Just a nit pick

Reply:
Thanks for the comments! This was something I did super quickly on my ipad and I easily missed these 😅